server/routes/oodler.js

```diff
@@ -68,28 +73,69 @@ let routes = [
   {
     method: 'GET',
     path: '/oodler/{id}',
-    handler: get
+    config: {
+      handler: get,
+      validate: {
+        params: {
+          id: Joi.string().token()
+        }
+      }
+    }
   },
   {
     method: 'POST',
     path: '/oodler',
-    handler: create
+    config: {
+      handler: create,
+      validate: {
+        payload: {
+          firstName: Joi.string().alphanum().required(),
+          lastName: Joi.string().alphanum().required(),
+          email: Joi.string().email().required(),
+          office: Joi.string().required(),
+          password: Joi.string().min(6).max(200).required(),
```

Comment:
Does this get `bcrypt`ed before being saved to the database?

Also, try to see this as an admin creating a user. Usually, the password is set automatically, and then the user changes it to whatever he wants via reset password dialog. A case can be made that the admin needs to do it, but just want you to be aware before getting this code out.